Add GSettings schema overrides for i3-GNOME-Flashback

deuill opened the pull request:
This commit compiles fixes made by SkyyySi, in setting sane
configuration defaults for GSettings/DConf, as used by GNOME and GNOME
Flashback.

For the most part, the defaults are meant to provide a correct
experience, especially where significant functionality is broken (such
as when a pre-defined desktop background is handled as an invisible
window by i3). Other, more opinionated configuration changes are left to
users.

deuill committed:
Add GSettings schema overrides for i3-GNOME-Flashback

This commit compiles fixes made by SkyyySi, in setting sane
configuration defaults for GSettings/DConf, as used by GNOME and GNOME
Flashback.

For the most part, the defaults are meant to provide a correct
experience, especially where significant functionality is broken (such
as when a pre-defined desktop background is handled as an invisible
window by i3). Other, more opinionated configuration changes are left to
users.

Co-authored-by: SkyyySi <rasti.b@protonmail.com>

deuill commented:
@SkyyySi I compiled effective changes from your fork and changed a few things, but have attributed you as a co-author; hope that's OK. What was changed:

- Wording in README was simplified, notes about fork removed.
- Button layout default has been changed to better conform with upstream default. I suspect the placement of these buttons is highly dependent on opinion, I've gone ahead and at least removed the minimize/maximize buttons set for GNOME Flashback, as those are definitely ineffectual. I personally just use `appmenu:`.

SkyyySi commented:
Using `appmenu:close` would probably be better here, as it prevents "Wait, where did the button go?!?" moments, especially for new users. Since most people using this have probably used i3 before, they will probably already have noticed that all other buttons don't do anything. So that should be the best overall compromise.

Btw, maybe there should be a quick explaination at the bottom of how it works or at least a link to the [official](https://help.gnome.org/admin/system-admin-guide/stable/overrides.html.en) [documentation](https://help.gnome.org/admin/system-admin-guide/stable/dconf-custom-defaults.html.en) (which I found to be super confusing, but it still lead me to the right track, alongside regolith-linux' settings overrides (which came in part from me 🙃).

PS: Thank's for merging it (or rather working on merging it, whatever)!

deuill committed:
README: Extend with details on GSettings defaults

This should help explain how defaults are set, and how to update these
if needed.

deuill commented:
Good point, I've updated the README with additional information here. Thanks for your contribution, again!
